cmd/kube-proxy/app/server_others.go

```diff
@@ -257,6 +257,10 @@ func newProxyServer(
 		proxymetrics.RegisterMetrics()
 	} else if proxyMode == proxyModeIPVS {
 		klog.V(0).Info("Using ipvs Proxier.")
+		// fast-fail if ipvsInterface is nil
+		if ipvsInterface == nil {
+			return nil, fmt.Errorf("unable to initialize IPVS interface")
```

Comment:
so this would still crash on first run, it would just do so with an error instead of a panic?

Reply:
Yeah, but this doesn't happen on every first run.
It's probably rare, for some wired reason (about kernel?)
Anyway, we should check the nil pointer first.